myuw/templates/calendar.html

```diff
@@ -14,17 +14,21 @@
 
```

Comment:
@fanglinfang My change on this file is on **line 12**. Changing the attribute `aria-labelledby`.